docs/notebooks/ExB_drift.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<p>I'm going to have to suggest writing it just as "ExB" - while this looks great in the notebook, this first heading is used in the HTML &lt;title&gt; tag, so if you take a look at the &lt;title&gt; of https://plasmapy--971.org.readthedocs.build/en/971/notebooks/ExB_drift.html, you'll see it breaks a little.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/971/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Thanks, I missed that. I'll change it in the next commit. </p><p> </p>